modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -311,11 +286,11 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
             ///////////////////////////////////////////////////////////////////////
             ////////////// make sure the advanced options are filled out///////////
             ///////////////////////////////////////////////////////////////////////
-            $candid      = trim($args['values']['CandID']);
-            $pscid       = trim($args['values']['PSCID']);
-            $visit_label = trim($args['values']['Visit_label']);
+            $candid      = trim($args['values']['candID']);
+            $pscid       = trim($args['values']['pSCID']);
```

Comment:
@jstirling91  The way filtering is implemented in `StaticDatatable` converts all headers to `camelCase`. So it requires users to pass `keys` in `camelCase` as well. 

Essentially, its a "bug" in `toCamesCase` function that does not have a special treatment for words in all caps